src/NHibernate/Transform/AliasToBeanResultTransformer.cs

```diff
@@ -48,22 +46,17 @@ public AliasToBeanResultTransformer(System.Type resultClass)
 			if (constructor == null && resultClass.IsClass)
 			{
 				throw new ArgumentException("The target class of a AliasToBeanResultTransformer need a parameter-less constructor",
-				                            "resultClass");
+											"resultClass");
 			}
 
-			propertyAccessor =
-				new ChainedPropertyAccessor(new[]
```

Comment:
This was the only usage of the class `ChainedPropertyAccessor`. With this change, this class as no more any purpose in NHibernate. This class should be marked with [Obsolete("Has no more usage and will be removed in a future version")].